configs/defs/fallback.textproto

```diff
@@ -1248,7 +1255,7 @@ benchmark_configs: {
       "re-running the file display command.\n"
     cis_benchmark: {
       profile_level: 1
-      severity: LOW
+      severity: HIGH
```

Comment:
Why the change in severity? Let's keep this LOW for now as our current policy is to consider config related security findings (as opposed to CVEs) low severity.